csrankings.csv

```diff
@@ -3994,6 +3995,7 @@ Edward P. Tighe,De La Salle University,https://www.dlsu.edu.ph/colleges/ccs/facu
 Edward Sciore,Boston College,http://www.cs.bc.edu/~sciore,NOSCHOLARPAGE
 Edward W. Felten,Princeton University,https://www.cs.princeton.edu/~felten,watx0IAAAAAJ
 Edward W. Knightly,Rice University,http://knightly.rice.edu,-0vALDMAAAAJ
+Edwin Olson,University of Michigan,http://april.eecs.umich.edu/,GwtVjKYAAAAJ

```

Comment:
Not the correct home page: https://april.eecs.umich.edu/people/ebolson/

Reply:
Thanks. Pushed a new commit.